packages/cli/lib/browser.js

```diff
@@ -12,12 +11,10 @@ module.exports = async (url, label, route, outputDirectory) => {
 
   const renderer = new Renderer(browser);
   const result = await renderer.serialize(url);
-
-  // need url: https://github.com/jsdom/jsdom/issues/2005
-  const dom = new JSDOM(result.content, { url });
```

Comment:
```
const dom = new JSDOM(``, {
  url: "https://example.org/",
  referrer: "https://example.com/",
  contentType: "text/html",
  includeNodeLocations: true,
  storageQuota: 10000000
});
```

> url sets the value returned by window.location, document.URL, and document.documentURI, and affects things like resolution of relative URLs within the document and the same-origin restrictions and referrer used while fetching subresources. It defaults to "about:blank".

Removing this line could potential cause issues related to path/url of  resources. I do not know where specifically this was an issue previously.

Reply:
Although from my cursory tests I haven't seen any immediate issues.

Reply:
As per the issue cited in the comment: https://github.com/jsdom/jsdom/issues/2005
```
<style type="text/css">
@import url("/css/blok_add.css");
</style>
```
Cannot be parsed without the correct url. Although I do not see that error serializing in puppeteer, maybe because we don't have any examples of relative css imports.

With jsdom:
> The problem can be solved by specifying the parameter url.
> 
> new JSDOM(html, {url: 'http://example.com'})
>

Reply:
https://github.com/hutchgrant/ssr-evergreen-demo/blob/b18f31038dd24fb1e003ffdb00a18a20429e1d6c/renderer.js#L31

From my previous SSR-demo which formed the basis of this code base, I previously has puppeteer serialize, then went used JSDOM to manipulate dom, then serialized again.

Perhaps thats why we kept it? In case we needed to change dom post-serialization via puppeteer?

Reply:
Gotta love unit tests :)

As part of the [PR that added that comment re: JSDOM](https://github.com/ProjectEvergreen/greenwood/pull/137/), we also added unit tests specifically for `@import`.  This was for issue #134 

All unit tests are passing and the website preview is working so I think all the right checks and balances are in place, IMO.

Reply:
In general, removing an extra translation layer seems like the right move architecturally too.  If we're already pre-rendering with a "real" browser, serializing it through fake one seems unnecessary.

Reply:
That said, if there are specific test cases we need to validate, please suggest them and I can look to see if they are being covered.